wiki/CreatingMissions.md

```diff
@@ -328,10 +326,22 @@ This specifies that the mission does not show up in the player's list of mission
 
 If a mission is marked with `priority`, only other "priority" missions can be offered alongside it.
 
-If a mission is marked with `minor`, it will be offered only if no other missions are being offered at the same time.  
+If a mission is marked with `minor`, it will be offered only if no other missions are being offered at the same time, including other minor missions. See the "order" attribute below to learn how to influence precedence if required.
 In general, any mission that starts a completely new mission string, and that could instead be offered at a later date, should be marked "minor." Missions continuing a string should not be marked "minor."
 
-Note that `priority` will only affect missions that offer from the spaceport.
+Note that `priority` will only affect missions that offer from the spaceport, outfitter or shipyard.
+
+```html
+order <number#>
+```
+
+This can optionally be used to reorder mission precedence. It defaults to 0 and can be negative. Not allowed for `job` missions, because those are sorted by user preference, nor for `boarding` or `assisting` missions.
```

Comment:
```suggestion
This can be used to reorder mission precedence. It defaults to 0 and is allowed to be negative. Does nothing for `job` missions (because those are sorted by user preference) nor for `boarding` or `assisting` missions.
```

Reply:
```suggestion
This can optionally be used to reorder mission precedence. It defaults to 0 and is allowed to be negative. Not allowed for `job` missions (because those are sorted by user preference) nor for `boarding` or `assisting` missions.
```